### ISSUE TITLE
seems some passwords are hard coded into endpoint checks

### ISSUE THREAD

**Issue by rstarmer** [digitalrebar/core#1255]
I changed my rebar user password, and yet I see:

//usr/local/entrypoint.d/23-wait-for-api-to-start.sh@3(source): rebar -U rebar -P rebar1 ping
2016/08/02 18:40:47 Could not connect to Rebar: Head https://172.17.0.11:3000/api/v2/digest: dial tcp 172.17.0.11:3000: getsockopt: connection refused

using the default password to try to query the API.

**Comment by galthaus** [comment]
get rid of the 3000.  We need to update docs and such.

**Comment by galthaus** [comment]
wait - hmmm - not sure if that will work.  We have a reverse proxy providing auth now.  You need to go through it normally.  Using internal container IPs is restricted unless you have cert/key pair that is authorized.

**Comment by rstarmer** [comment]
well, the issue was in looking at the log files of the api container which
i restarted, but _after_ I had changed the rebar password on my system.
I.e., this is hard coded in the dr_rebar_api container at a minimum...

Robert

**Comment by galthaus** [comment]
Yeah - we are in the process of trying to convert the bring-up sequences to use a bootstrap user without a password.  We aren't quite there yet.  I think @VictorLowther is working on it.

**Comment by rstarmer** [comment]
Ok, I will work around this in the near term then...

Robert
